pkg/agent/openflow/client.go

```diff
@@ -290,14 +292,22 @@ func (c *client) IsConnected() bool {
 
 // addFlows installs the flows on the OVS bridge and then add them into the flow cache. If the flow cache exists,
 // it will return immediately, otherwise it will use Bundle to add all flows, and then add them into the flow cache.
-// If it fails to add the flows with Bundle, it will return the error and no flow cache is created.
-func (c *client) addFlows(cache *flowCategoryCache, flowCacheKey string, flows []binding.Flow) error {
-	_, ok := cache.Load(flowCacheKey)
-	// If a flow cache entry already exists for the key, return immediately. Otherwise, add the flows to the switch
-	// and populate the cache with them.
-	if ok {
-		klog.V(2).Infof("Flows with cache key %s are already installed", flowCacheKey)
-		return nil
+// If it fails to add the flows with Bundle, it will return the error and no flow cache is created. If the force parameter
+// is true, flows will be added regardless of flow cache.
+func (c *client) addFlows(cache *flowCategoryCache, flowCacheKey string, flows []binding.Flow, force bool) error {
+	if !force {
+		_, ok := cache.Load(flowCacheKey)
+		// If a flow cache entry already exists for the key, return immediately. Otherwise, add the flows to the switch
+		// and populate the cache with them.
+		if ok {
+			klog.Infof("Flows with cache key %s are already installed", flowCacheKey)
```

Comment:
it's too verbose to have this as default level log

Reply:
This should be v=2. This is changed during debugging and I forgot to change it back.